website/content/commands/rtt.mdx

```diff
@@ -9,12 +9,22 @@ description: |
 
 Command: `consul rtt`
 
+Corresponding HTTP API Endpoints: [\[GET\] /v1/coordinate/datacenters](https://www.consul.io/api-docs/coordinate#read-wan-coordinates), [\[GET\] /v1/coordinate/nodes](https://www.consul.io/api-docs/coordinate#read-lan-coordinates-for-all-nodes)
+
 The `rtt` command estimates the network round trip time between two nodes using
 Consul's network coordinate model of the cluster.
 
 See the [Network Coordinates](/docs/internals/coordinates) internals guide
 for more information on how these coordinates are computed.
 
+The table below shows this command's [required ACLs](/api#authentication). Configuration of
+[blocking queries](/api/features/blocking) and [agent caching](/api/features/caching)
+are not supported from commands, but may be from the corresponding HTTP endpoint.
+
+| ACL Required |
+| ------------ |
+| `node:read`  |
```

Comment:
According to the HTTP API docs, only reading LAN coordinates requires `node:read`. It seems that reading WAN coordinates requires no ACL permissions. Not sure how best to note that here. Thoughts? (maybe the footnote approach used in other places?)